lib/widgets/package_list_view.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:pub_client/pub_client.dart';
+import 'package:tavern/widgets/package_tile.dart';
+
+class PackageListView extends StatefulWidget {
```

Comment:
why is this Stateful?

Reply:
It may have had a scroll controller in there. If it doesn't now, it's because it got refactored into a different widget and I forgot. I'll fix this as well

Reply:
Shouldn't this be Stateless now?